.github/workflows/ci.yml

```diff
@@ -22,12 +22,8 @@ jobs:
 
     - name: Set Chrome Version
       run: |
-        CHROMEVER="$(chromedriver --version | cut -d' ' -f2)"
+        export CHROMEVER="$(chromedriver --version | cut -d' ' -f2)"
```

Comment:
@keithamus and @koddsson this incorporates the CI configuration you two introduced in https://github.com/hotwired/turbo/pull/551/commits/b1cbe123895259408d35b1c918ab2e9a51ba6683 to support development-environment configurations.

It's still an incomplete developer experience and requires an awareness of your local chrome driver version.

On top of that, I'm not sure how to programmatically set a GitHub actions environment variable in this way, `export` doesn't seem to cut it.

Reply:
> On top of that, I'm not sure how to programmatically set a GitHub actions environment variable in this way, export doesn't seem to cut it.

Evidently, piping an assignment into `$GITHUB_ENV` does the trick!